lib/backend/k8s/conversion/conversion.go

```diff
@@ -196,19 +209,20 @@ func (c Converter) PodToWorkloadEndpoint(pod *kapiv1.Pod) (*model.KVPair, error)
 	// Create the key / value pair to return.
 	kvp := model.KVPair{
 		Key: model.ResourceKey{
-			Name: pod.ObjectMeta.Name,
-			Kind: apiv2.KindWorkloadEndpoint,
+			Name:      wepName,
+			Namespace: pod.ObjectMeta.Namespace,
+			Kind:      apiv2.KindWorkloadEndpoint,
 		},
 		Value: &apiv2.WorkloadEndpoint{
 			ObjectMeta: metav1.ObjectMeta{
-				Name:      pod.ObjectMeta.Name,
+				Name:      wepName,
 				Namespace: pod.ObjectMeta.Namespace,
 				Labels:    labels,
 			},
 			Spec: apiv2.WorkloadEndpointSpec{
 				Orchestrator:  "k8s",
 				Node:          pod.Spec.NodeName,
-				Workload:      workload,
+				Pod:           pod.Name,
```

Comment:
but the resource is namespaced so why does this name need to be namespaced?  I don't think it does

I think the converter that maps this to the v1 model needs to prepend the namespace to the name perhaps with a / -- since Felix is not restrictive on the use of /

Reply:
make sense.